src/template/agentPagesAgentBrief.php

```diff
@@ -26,6 +26,18 @@
 
 	<div class="wolfnet_aoBody">
 		
+	<?php
+	 	$agentArray = preg_split("/\//", $agentLink);
+		$agentName = $agentArray[count($agentArray) - 2];
```

Comment:
@jonlipp This line is pretty fragile, because it depends upon the array always having a minimum length. The URI that the array is based on is created in the AgentPagesHandler's `buildLinkToAgent()` and `buildLink` functions. I recommend moving this whole block of new php code over to one of those functions.

For an example of how this could be fragile, see the "Agent/Office Pages Preview" in the admin's WolfNet: Appearance page. If this new code were to be used in the AgentPagesHandler's link-builder functions, the preview in the admin would work again.